src/web-ui/src/authenticated/Profile.vue

```diff
@@ -67,22 +73,27 @@
                   Please enter a valid email address for shipping updates.
                 </div>
               </div>
-
             </form>
-            <button class="btn btn-primary" v-on:click="saveChanges"> <i class="fas fa-spinner fa-spin" v-if="saving"></i> Save Changes </button>
+            <button class="btn btn-primary" v-on:click="saveChanges"><i class="fas fa-spinner fa-spin"
+                                                                        v-if="saving"></i> Save Changes
+            </button>
           </div>
-
         </div>
+        <h4>Location Demo</h4>
+        <hr/>
+        <router-link type="button" class="btn btn-primary" :to="{name:'Collections'}">
```

Comment:
The profile page doesn't exist anymore. You could add a link in `UserDropdown` instead.